### ISSUE TITLE
'SurfacePro4 and Surface Pro 4 (2)' controls define as a button but visually looks like an Image on 'Receipt card' content.

### ISSUE THREAD

**Issue by JasonAment** [microsoft/AdaptiveCards#2242]
Please review bug repro steps and video at the following link:
https://github.com/Microsoft/BotFramework-WebChat/issues/1368

**Comment by shalinijoshi19** [comment]
We need to follow up here with our Accessibility team on this one since it's unclear how AC should be fixing this. @dclaux / @matthidinger for FYI.

**Comment by RebeccaAnne** [comment]
We believe this should be working now, please reactivate if this is still an issue for you.
